packages/yourloops/components/statistics/patient-statistics.tsx

```diff
@@ -61,7 +61,7 @@ export const PatientStatistics: FunctionComponent<PropsWithChildren<PatientStati
         days={dataUtil?.days ?? 0}
       />
       <Divider sx={{ marginBlock: theme.spacing(1), backgroundColor: theme.palette.grey[600] }} />
-      { cbgSelect && <SensorUsageStat total={total} sensorUsage={sensorUsage}/>}
+      {cbgSelect && <SensorUsageStat dataUtil={dataUtil} />}
       {children}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏Just missing a divider here 
```suggestion
      {cbgSelect && <SensorUsageStat dataUtil={dataUtil} />}
      <Divider sx={{ marginBlock: theme.spacing(1), backgroundColor: theme.palette.grey[600] }} />
      {children}
```